mimic/rest/maas_api.py

```diff
@@ -720,6 +766,43 @@ def delete_alarm(self, request, tenant_id, entity_id, alarm_id):
         self._audit('alarms', request, tenant_id, status)
         return ''
 
+    @app.route('/v1.0/<string:tenant_id>/entities/<string:entity_id>/test-alarm', methods=['POST'])
+    def test_alarm(self, request, tenant_id, entity_id):
+        """
+        Test an alarm, pulling responses from the simulated test-alarm queue.
```

Comment:
If I were reading this docstring trying to understand where the "simulated test-alarm queue" was set up, I'd be confused.  Can you include a reference for folks to understand how said queue is populated?  "``` :obj:`a.python.name` ```" is a good way to do this if you are just referring to a Python API.